Client/Frontend/Strings.swift

```diff
@@ -184,3 +184,8 @@ extension Strings {
     public static let UnableToAddPassErrorMessage = NSLocalizedString("AddPass.Error.Message", value: "An error occured while adding the pass to Passbook. Please try again later.", comment: "Text of the 'Add Pass Failed' alert")
     public static let UnableToAddPassErrorDismiss = NSLocalizedString("AddPass.Error.Dismiss", value: "OK", comment: "Button to dismiss the 'Add Pass Failed' alert")
 }
+
+// Top Tabs
+extension Strings {
+    public static let TopTabsNewTabTitle = NSLocalizedString("TopTabs.Tiltle.NewTab", value: "New Tab", comment: "The title displayed in the tab when opening a new tab.")
```

Comment:
I assume this string was refactored out of an existing button since we already had the text 'New Tab' available. We just can't add any additional strings with a new value for NSLocalizedString.